pyproject.toml

```diff
@@ -51,10 +51,11 @@ lint = [
 ]
 test = ["pytest", "pytest-cov", "coverage", "jupyter", "ipython < 8.23"]
 graph = ["networkx"]
+varifold = ["pykeops"]
 autograd = ["autograd >= 1.3"]
 pytorch = ["torch >= 1.9.1"]
 backends = ["geomstats[autograd, pytorch]"]
-opt = ["geomstats[graph]"]
+opt = ["geomstats[graph, varifold]"]
```

Comment:
name it "pykeops" instead of varifold, since we use the name of packages in the pyproject.toml (not the name of the math...)